apps/design-land/src/app/foundations/color/color.component.html

```diff
@@ -124,3 +124,18 @@ <h2>Decorative palettes</h2>
 		<div style="background-color: #07230d;"><span class="dl-color__hex">100</span><span class="dl-color__hue">#07230d</span></div>
 	</div>
 </div>
+
+<h2>Autogenerated palettes</h2>
+<div class="dl-color__decorative-grid">
+  @for (color of palette$ | async | keyvalue; track color) {
+    @if (color.value.kind === GROUP) {
+        <div class="dl-color__palette brand-{{color.key}}">
```

Comment:
`{{color.key}}` needs to be lowercased, otherwise the class doesn't get styled since we always use lowercase in scss files.

<img width="311" alt="Screen Shot 2024-07-16 at 1 20 38 PM" src="https://github.com/user-attachments/assets/f095a924-e439-4875-a1f4-6e6aa69a12d0">

Reply:
done